quic/api/Observer.h

```diff
@@ -72,6 +72,21 @@ class InstrumentationObserver {
         lastAckedPacketInfo;
   };
 
+  struct ObserverSpuriousLossEvent {
```

Comment:
Since this is already in the `InstrumentationObserver` class, let's drop "Observer" from the title. Can just call it `SpuriousLossEvent`. I will rename the other one (loss event) when I get a chance.